rusk/tests/common/state.rs

```diff
@@ -130,10 +134,10 @@ pub fn generator_procedure(
     )
     .expect("valid block");
 
-    let verify_output = rusk.verify_state_transition(&block)?;
+    let verify_output = rusk.verify_state_transition(&block, &[])?;
     info!("verify_state_transition new verification: {verify_output}",);
 
-    let (accept_txs, accept_output) = rusk.accept(&block)?;
+    let (accept_txs, accept_output) = rusk.accept(&block, &[])?;
```

Comment:
You should have the same voters here too (otherwise a mismatch state happen)

Reply:
Indeed.
Now the voters slice is wrapped in an Option and None is used in any unit tests.

Reply:
Why don't you use a empty slice? 
No need for an option there

Reply:
If it's an empty slice, we return an error